tensorrt_llm/sampling_params.py

```diff
@@ -280,13 +282,6 @@ def __post_init__(self):
 
         self.best_of = self.best_of or self.n
 
-        if (not self.use_beam_search and self.n < self.best_of
```

Comment:
I'm not exactly sure what this is, still confirming w/ Xiaowei this is no longer needed.